synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -0,0 +1,154 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file docks/dock_timetrack2.cpp
+**	\brief Dock to displaying layer parameters timetrack
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**  ......... ... 2020 Rodolfo Ribeiro Gomes
+**
+**	This package is free software; you can redistribute it and/or
+**	modify it under the terms of the GNU General Public License as
+**	published by the Free Software Foundation; either version 2 of
+**	the License, or (at your option) any later version.
+**
+**	This package is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+**	General Public License for more details.
+**	\endlegal
+*/
+
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include "dock_timetrack2.h"
+
+#include <widgets/widget_timetrack.h>
+#include <gui/canvasview.h>
+#include <gui/localization.h>
+
+#include <synfig/general.h>
+
+#endif
+
+using namespace studio;
+
+Dock_Timetrack2::Dock_Timetrack2()
+	: Dock_CanvasSpecific("timetrack2", _("Timetrack"), Gtk::StockID("synfig-timetrack")),
```

Comment:
I suggest to change into this:
```
Dock_CanvasSpecific("timetrack", _("Timetrack"), Gtk::StockID("synfig-timetrack")),
```

...and change initialization of old timetrack (in `dock_timetrack.cpp`) to this:
```
Dock_CanvasSpecific("timetrack-old", _("Timetrack (old)"), Gtk::StockID("synfig-timetrack")),
```

Reply:
Done